tests/components/deconz/test_gateway.py

```diff
@@ -209,12 +214,10 @@ async def test_gateway_device_configuration_url_when_addon(
     )
 
 
-async def test_connection_status_signalling(
-    hass: HomeAssistant, aioclient_mock: AiohttpClientMocker, mock_deconz_websocket
-) -> None:
-    """Make sure that connection status triggers a dispatcher send."""
-    data = {
-        "sensors": {
+@pytest.mark.parametrize(
```

Comment:
This is ok but I don't understand the need reason to parametrize here when there is only one configuration in the list.

Reply:
To keep the test as clean as possible